### PR TITLE
Change the camera layout.

### PR DESCRIPTION
**Pull request by affan00**
@azharmalik3 Change the cameras 'created at' table on the first column and set the sorting to the newest first.

### CODE REVIEW

File: app/controllers/cameras_controller.rb [hunk 1]
```diff
@@ -48,9 +48,9 @@ def merge
   def load_cameras
     col_for_order = params[:order]["0"]["column"]
     order_for = params[:order]["0"]["dir"]
-    condition = "lower(cameras.exid) like lower('%#{params[:fquery]}%') OR lower(cameras.name) like lower('%#{params[:fquery]}%') OR 
+    condition = "lower(cameras.exid) like lower('%#{params[:fquery]}%') OR lower(cameras.name) like lower('%#{params[:fquery]}%') OR
```
Comment (marcoherbst) [inline app/controllers/cameras_controller.rb]: CamerasController#load_cameras calls 'params[:fquery]' 6 times

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/assets/javascripts/admin/cameras.js.coffee [hunk 2]
```diff
@@ -35,29 +35,29 @@ initializeDataTable = ->
...
-        {data: "2", "width": "125px" },
-        {data: "3", "width": "90px" },
-        {data: "4", "width": "70px" },
-        {data: "5", "width": "75px" },
-        {data: "6", "width": "135px" },
-        {data: "7", "width": "100px" },
-        {data: "8", "width": "110px" },
-        {data: "9", "width": "100px" },
-        {data: "10", "width": "130px" },
+        {data: "0", "width": "105px", "orderable": true, "sType": "uk_datetime" },
```
Comment (marcoherbst) [inline app/assets/javascripts/admin/cameras.js.coffee]: Line exceeds maximum allowed length

**Comment by marcoherbst** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 possible new issues (including those that may have been commented here).
* 2 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/evercam/evercam-admin/pulls/264.

**Comment by ijunaid8088** [comment]
Fix hound only if there any.. Leave Ebert. @affan00

**Comment by affan00** [comment]
Ok @ijunaid8088 thanks

**Pull request merged**